docs/sections/user_guide/yaml/components/fv3.rst

```diff
@@ -84,7 +84,7 @@ Supports ``base_file:`` and ``update_values:`` blocks (see the :ref:`updating_va
 files_to_copy:
 ^^^^^^^^^^^^^^
 
-Defines files to be copied to the run directory. Keys in the ``files_to_copy:`` YAML map specify destination paths relative to the run directory, and values specify source paths. Both keys and values may contain Jinja2 variables/expressions using a ``cycle`` variable, which is a Python ``datetime`` object corresponding to the FV3 cycle being run. This supports specification of cycle-specific filenames/paths. For example, a key-value pair
+See :ref:`this page <files_yaml>` for details. For the ``fv3`` driver, both keys and values may contain Jinja2 variables/expressions using a ``cycle`` variable, which is a Python ``datetime`` object corresponding to the FV3 cycle being run. This supports specification of cycle-specific filenames/paths. For example, a key-value pair
```

Comment:
Since YAML blocks with these semantics are now used both by the `fv3` driver and by the `file` tool (and will undoubtedly be used for other purposes in the future), I've factored the description out into its own file and have linked to it where needed.